radicle-cli/src/commands/node/control.rs

```diff
@@ -0,0 +1,51 @@
+use radicle::node::{Address, Handle as _, NodeId};
+use radicle::Node;
+
+use crate::terminal as term;
+
+pub fn start() -> anyhow::Result<()> {
+    todo!()
+}
+
+pub fn stop(node: Node) -> anyhow::Result<()> {
+    let spinner = term::spinner("Stopping the node...");
+    if let Err(err) = node.shutdown() {
+        spinner.finish();
+        term::warning(&format!("Error occurred while shutting down node: {err}"));
+    } else {
+        spinner.finish();
+        term::success!("Successfully shut down the node")
+    }
+    Ok(())
+}
+
+pub fn connect(node: &mut Node, nid: NodeId, addr: Address) -> anyhow::Result<()> {
+    let spinner = term::spinner("Connecting to node...");
```

Comment:
I'd either just say `Connecting...` or be a bit more specific and say `Connecting to {nid}@{addr}...`